solutions/tf-extension-da/README.md

```diff
@@ -0,0 +1,50 @@
+# Terraform based "extension" sample deployable architecture - Serving static websites with IBM Cloud Object Storage
+
+This architecture supports extending an existing Object Storage bucket with the configuration required to host a static website. The solution will configure the following things:
+- enables public access for the provided Object Storage bucket
+- adds the website configuration to host a static website
+
```

Comment:
```suggestion
# Terraform-based deployable architecture with dependencies sample- Serving static websites with IBM Cloud Object Storage

This architecture supports extending an existing Object Storage bucket with the configuration required to host a static website. Because this deployable architecture has an `install_type` set to `extension` in the `ibm_catalog.json`, you must also specify a dependency, which in this case is an existing Object Storage bucket. By using this type of deployable architecture install type, the IBM Cloud catalog will automatically detect whether  an Object Storage bucket exists in the account, and if it does, then this option will be available to be deployed to further extend it.

The solution will configure the following things:
- enables public access for the provided Object Storage bucket
- adds the website configuration to host a static website

```

There is likely a better way to describe this extension type/dependency scenario but I thought I'd start it with this suggestion. @SirSpidey Maybe you can make this better. I'm trying to get at this is different than setting a dependency in the manifest and just being a fullstack DA that has a marked prereq.

Reply:
@mekaufman what you have proposed here is not true. The dependency is not any COS bucket in the account, the dependency is on another DA (in this case its on the COS replication fullstack DA I also create in this repo). So the check that occur is checking that the dependant DA exists in a schematics workspace in the account. If it does not, the user gets a warning in the UI, but it does not prevent them from deploying the extension DA anyway (since maybe they didn't deploy it using schematics)

Reply:
oh, interesting... I set up time Monday because I'm not sure I fully get the interaction pattern in the catalog, so maybe we can work this through together.